Revise AAC templates for consistency and latest guidance (#48)
MicrosoftDocs/content-templates: templates/org/aac/aac-article-title-content.md

<!-- 
- Don't add metadata to this Markdown file. Use the browser header template to create a YAML file that contains your metadata. 
- Add a brief introductory paragraph with no heading. 
-->

## Architecture

<!-- Use the following format to add your image file:-->

> :::image type="content" border="false" source="./images/<file-name>.png" alt-text="Diagram that shows the <solution name> architecture." lightbox="./images/<file-name>.png":::

Reviewer: ```suggestion
:::image type="complex" source="./images/<file-name>.png" alt-text="Diagram that shows the <solution name> architecture." lightbox="./images/<file-name>.png":::
    <long description here>
:::image-end:::
```

<!-- The following link will work after the AAC team uploads your Visio or PowerPoint file to the Azure CDN. -->

*Download a [Visio file](https://arch-center.azureedge.net/<file-name>.vsdx) of this architecture.*

### Dataflow

<!-- If your scenario doesn't include data, title this section "Workflow". -->

The following dataflow corresponds to the previous diagram:

<!-- Add a numbered list for the steps in your architecture diagram. -->

### Components

<!-- 
- Add a bulleted list of all components in the architecture. 

> - Where possible, link to the component's Well-Architected Framework service guide. Alternatively, link to the product page.

Reviewer: ```suggestion
- Where possible, link to the component's Well-Architected Framework service guide. Alternatively, link to the product's documentation page.
```

-->

### Alternatives

> <!-- List alternative Azure services or architectures for this solution. -->

Reviewer: ```suggestion
<!-- List alternative Azure services or approaches for this solution. -->
```

### Reliability

<!--REQUIRED STATEMENT: Include the following statement to introduce the section:-->

Reliability ensures your application can meet the commitments you make to your customers. For more information, see [Design review checklist for Reliability](/azure/well-architected/reliability/checklist).

<!-- Explain resiliency and availability considerations. -->

### Security

<!--REQUIRED STATEMENT: Include the following statement to introduce the section:-->

Security provides assurances against deliberate attacks and the misuse of your valuable data and systems. For more information, see [Design review checklist for Security](/azure/well-architected/security/checklist).

<!-- Explain security, identity, and data sovereignty considerations. -->

### Cost Optimization

<!--REQUIRED STATEMENT: Include the following statement to introduce the section:-->

Cost Optimization focuses on ways to reduce unnecessary expenses and improve operational efficiencies. For more information, see [Design review checklist for Cost Optimization](/azure/well-architected/cost-optimization/checklist).

<!-- 
- Explain Cost Optimization considerations. 

> - Link to the pricing calculator (https://azure.microsoft.com/pricing/calculator). 

Reviewer: ```suggestion
- Link to the pricing calculator (https://azure.microsoft.com/pricing/calculator) with this architecture pre-populated.
```

- Include the major cost-driving components, a typical scale or throughput, and recommended SKUs. 
-->

### Operational Excellence

<!--REQUIRED STATEMENT: Include the following statement to introduce the section:-->

Operational Excellence covers the operations processes that deploy an application and keep it running in production. For more information, see [Design review checklist for Operational Excellence](/azure/well-architected/operational-excellence/checklist).

<!-- Explain Operational Excellence considerations. -->

### Performance Efficiency

<!--REQUIRED STATEMENT: Include the following statement to introduce the section:-->

Performance Efficiency refers to your workload's ability to scale to meet user demands efficiently. For more information, see [Design review checklist for Performance Efficiency](/azure/well-architected/performance-efficiency/checklist).

<!-- Explain key performance considerations, beyond the typical. -->

## Contributors

<!-- This section is expected but optional if the contributors prefer to omit it. -->

*Microsoft maintains this article. The following contributors wrote this article.*

> Principal authors: 

Reviewer: ```suggestion
Principal authors:
```

<!-- List the primary authors alphabetically by last name. -->

> - [Author 1 Name](https://linkedin.com/in/ProfileURL) | Title, such as "Cloud Solution Architect"
> - [Author 2 Name](https://linkedin.com/in/ProfileURL) | Title, such as "Cloud Solution Architect"

Reviewer: ```suggestion
- [Author 1 Name](https://www.linkedin.com/in/ProfileURL/) | Title, such as "Cloud Solution Architect"
- [Author 2 Name](https://www.linkedin.com/in/ProfileURL/) | Title, such as "Cloud Solution Architect"
```

> Other contributors: 

Reviewer: ```suggestion
Other contributors:
```

<!--
- This section is optional. 
- List contributors and technical reviewers. 
-->

> - [Contributor 1 Name](https://linkedin.com/in/ProfileURL) | Title, such as "Cloud Solution Architect"
> - [Contributor 2 Name](https://linkedin.com/in/ProfileURL) | Title, such as "Cloud Solution Architect"

Reviewer: ```suggestion
- [Contributor 1 Name](https://www.linkedin.com/in/ProfileURL/) | Title, such as "Cloud Solution Architect"
- [Contributor 2 Name](https://www.linkedin.com/in/ProfileURL/) | Title, such as "Cloud Solution Architect"
```
